clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -7222,6 +7222,11 @@ void EventAPICallRule::handleEventRecordWithProfilingEnabled(
   auto IndentLoc = CE->getBeginLoc();
   auto &SM = DpctGlobalInfo::getSourceManager();
 
+  if (const auto *UO = dyn_cast<UnaryOperator>(StreamArg->IgnoreImpCasts())) {
```

Comment:
Is it better to compare the precedence of `*` and `->`?

Reply:
Yes, updated.